conf/messages.en

```diff
@@ -557,3 +557,31 @@ expensesStart.details.content6 = Or
 expensesStart.details.content7 = where you enter an amount for each separate expenses type.
 expensesStart.details.content7.link.text = Individual expenses (opens in new tab)
 expensesStart.details.content7.link.href = https://www.gov.uk/government/publications/self-assessment-uk-property-sa105/uk-property-notes-2022#property-expenses
+
+consolidatedExpenses.title = Consolidated expenses
+consolidatedExpenses.heading = Consolidated expenses
+consolidatedExpenses.details.heading = Understanding consolidated property expenses
+consolidatedExpenses.details.line1.individual = If your total property income before expenses is less than £85,000, you can just add up all of these expenses and enter the total amount:
+consolidatedExpenses.details.line1.agent = If your client's total property income before expenses is less than £85,000, add up all of these expenses and enter the total amount:
```

Comment:
replace ' with ’
I have done a find and replace, but it doesn't show on the page if you use ' instead of ’